pipelines/internal/update-fbc-catalog/README.md

```diff
@@ -7,10 +7,14 @@ Tekton pipeline add/update FBC fragments to the FBC catalog by interacting with
 | Name                    | Description                                                                 | Optional | Default value       |
 |-------------------------|-----------------------------------------------------------------------------|----------|---------------------|
 | iibServiceAccountSecret | Secret containing the credentials for IIB service                           |   yes    | iib-service-account |
+| publishingCredentials   | Publishing credentials used to fetch the image config                       |   yes    | catalog-publishing-credentials |
```

Comment:
Can we say "secret containing..." to make it clearer what is expected?

Reply:
fixed.